app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -2629,6 +2630,23 @@ class PluginApi {
       .lookup("service:admin-custom-user-fields")
       .addProperty(userFieldProperty);
   }
+
+  /* Generates a visual feedback indicator for a specific post action.
+   *
+   * For example: when you click the "chain" on a post you see a visual indicator it
+   * was copied, with a checkmark and text that hovers above it saying it was copied.
```

Comment:
I think it will also be helpful to provide an example with code as to how the plugin-api can be used because I feel like this is not clear at the moment without having to look into the `postActionFeedback` function.